devops/ansible/roles/girder/tasks/pip.yml

```diff
@@ -9,6 +9,16 @@
         - pip
         - setuptools
 
+    - name: Force install of requests and six
+      pip:
+        name: "{{ item }}"
+        extra_args: "-I"
+        virtualenv: "{{ girder_virtualenv }}"
+        virtualenv_python: "{{ girder_python | default(ansible_python.executable) }}"
+      with_items:
+        - six
+        - requests
+
```

Comment:
I think this block is not actually needed as things seem to work in the virtualenv case.

Reply:
Can you remove this block and replace the other `-I` with `--ignore-installed`?

Reply:
Sure

Reply:
Done